src/bootstrap/tool.rs

```diff
@@ -260,6 +260,7 @@ tool!(
     BuildManifest, "src/tools/build-manifest", "build-manifest", Mode::Libstd;
     RemoteTestClient, "src/tools/remote-test-client", "remote-test-client", Mode::Libstd;
     RustInstaller, "src/tools/rust-installer", "fabricate", Mode::Libstd;
+    RustdocJS, "node", "node", Mode::Tool;
```

Comment:
You almost certainly want these arguments to be closer to the above. Something like `RustdocJs, "rustdoc-js", "js-tests", Mode::Tool`.

But frankly I'm not entirely certain what this tool is doing either way -- you'd probably actually not want it, or at least it seems like at the very least it shouldn't be generated by the macro. If this doesn't make sense, explaining why it's here would help.

Reply:
I just tried my best understanding what was going on in here. My goal is to test the search in rust documentation. So I need to run this tool once the documentation has been generated. To run it, I need nodejs and I don't really know how to pick up the currently installed one.